iqair.py

```diff
@@ -45,7 +52,12 @@ def __fetch_data(self, location: Location) -> json:
                 logging.error("Invalid request; possible reason: API change.")
                 exit(1)
             if e.status == 429:
-                logging.warn("Being throttled. Increase value of main.loop_seconds in config.yaml if non-zero.")
+                logging.warn("Being throttled. See README.md for options.")
+                self._throttle_count += 1
+                if self._throttle_count <= self._throttle_retry_max_count:
+                    time.sleep(1+self._throttle_count*random.randint(1, self._throttle_retry_wait_seconds))
```

Comment:
Minimum time for the first throttle count is 2 seconds.  This is way too little.  I am not sure how critical the data is, but I would wait at least couple minutes.  Even 10 minutes.

Reply:
I'm fine with that.